### PR TITLE
Quintanalibre lime 2020.1

### PR DESCRIPTION
**Pull request by spiccinini**
Update the config to use the latest features of libremesh generic configurations.

### CODE REVIEW

**Comment by ilario** [comment]
Please consider #66 also :)

File: quintanalibre.org.ar/generic/files/etc/config/lime-community [hunk 1]
```diff
@@ -38,3 +38,22 @@ config lime wifi
...
+config generic_uci_config system_opts
+	list uci_set "system.@system[0].timezone=ART3"
+	list uci_set "system.@system[0].log_ip=10.5.1.4"
+
+config generic_uci_config location
+	list uci_set "location.settings=location"
+	list uci_set "location.settings.community_latitude=-31.80461"
+	list uci_set "location.settings.community_longitude=-64.41609"
+
+config generic_uci_config deferable_reboot
```
Comment (germanferrero) [inline quintanalibre.org.ar/generic/files/etc/config/lime-community]: From this test: [test lime.config has precedence over config/deferable-reboot](https://github.com/libremesh/lime-packages/blob/ab721954269ffc9699882f7434d73d73f2d41556/packages/deferable-reboot/tests/test_deferable_reboot.lua#L29) I understand that the [default config at lime-defaults](https://github.com/libremesh/lime-packages/blob/26a5504caa7f20d41edcdd995805fe1c8140f75f/packages/lime-system/files/etc/config/lime-defaults#L14) will take precedence making this config useless.

Also, is this change in reboot time an improvement or a refactor?

File: quintanalibre.org.ar/generic/files/etc/uci-defaults/97-dropbear [hunk 2]
```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-# Fix dropbear authorized keys permissions
-chmod 0600 /etc/dropbear/authorized_keys
-chmod 0700 /etc/dropbear
```
Comment (germanferrero) [inline quintanalibre.org.ar/generic/files/etc/uci-defaults/97-dropbear]: We have lost the permission setup for the /etc/dropbear directory, (not for authorized_keys). Which AFAIU is not a big issue. But I raise my hand to have a second opinion

File: quintanalibre.org.ar/generic/files/etc/config/lime-community (hunk 1 above)
Comment (spiccinini) [reply quintanalibre.org.ar/generic/files/etc/config/lime-community]: Thanks, good catch! I was thinking on removing the deferable lime.config option  now that we can use the generic configs.

The idea is to maintain the original functionality that was checking each 3 hours and reboot if not internet. So the idea here is to move to check each 10m if there is internet from 3hs of uptime. It is not exactly the same as before. Maybe something better is to cheack "each hour" after the first 3 hours?

**Comment by spiccinini** [comment]
TODO: move authorized keys and other files to lime-assets

**Comment by germanferrero** [comment]
> TODO: move authorized keys and other files to lime-assets

Done!

**Pull request merged**